misago/legal/models.py

```diff
@@ -1 +1,127 @@
 # Empty models file that triggers migrations for this app
+from django.db import models
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+import datetime
+
+
+class NoActiveLegalAgreement(ValidationError):
+    pass
+
+
+class BaseModel(models.Model):
+    created = models.DateTimeField(auto_now_add=True, editable=False)
```

Comment:
lets use `default=timezone.now` for consistency with rest of codebase, and drop `editable` as we don't use or really support Django admin for Misago's models. Date/DateTime fields should also have `_on` or `_at` prefix whichever is more correct.